<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenecutOPWidget</class>
 <widget class="QWidget" name="GenecutOPWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Login form</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leEmail">
         <property name="text">
          <string>a.seryakov@unipro.ru</string>
         </property>
         <property name="placeholderText">
          <string>username@server.com</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePasword">
         <property name="text">
          <string>1234567</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbShowPass">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Show password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbLogin">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbForgot">
         <property name="text">
          <string>Forgot password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRegister">
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbLoginWarning">
         <property name="styleSheet">
          <string notr="true">font-weight: 600;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lbWelcome">
         <property name="text">
          <string>Welcome, </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbTest">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbTestInfo">
         <property name="styleSheet">
          <string notr="true">font-weight: 600;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>491</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Register form</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leEmailNew">
         <property name="placeholderText">
          <string>username@server.com</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePasswordNew">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePasswordConformationNew">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password confirmation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leFirstName">
         <property name="placeholderText">
          <string>First name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leLastName">
         <property name="placeholderText">
          <string>Last name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbRegisterWarning">
         <property name="styleSheet">
          <string notr="true">font-weight: 600;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRegisterNew">
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>351</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
